src/absil/il.fsi

```diff
@@ -1690,8 +1696,9 @@ type IPrimaryAssemblyTraits =
 // ==================================================================== *)
 
 /// A table of common references to items in primary assebly (System.Runtime or mscorlib).
-/// If you have already loaded a particular version of system runtime assembly you should reference items via an ILGlobals for that particular 
-/// version of system runtime assembly built using mkILGlobals. 
+/// If you have already loaded a particular version of system runtime assembly you should 
+/// reference items via an ILGlobals for that particular version of system runtime assembly 
```

Comment:
reference items from it via an ILGlobals for that specific version built using mkILGlobals.